app/src/pages/common/settingsPage/ruleList/listItem/listItem.jsx

```diff
@@ -28,10 +29,12 @@ export const ListItem = ({ item, getListItemContentData, ...rest }) => (
   <div className={cx('list-item')}>
     <ControlPanel item={item} {...rest} />
     <div className={cx('data')}>
-      {getListItemContentData(item).map((itemData, index) => (
-        // eslint-disable-next-line react/no-array-index-key
-        <ItemContent key={`${itemData.key}_${index}`} data={itemData} />
-      ))}
+      <ScrollWrapper autoHeight autoHeightMax={90} hideTracksWhenNotNeeded>
```

Comment:
Please make the scrolling option customizable because we don't need this functionality for `notifications` and `patterns` (it also uses the `RuleList` component).